Sources/HaCWebsiteLib/Events/EventServer.swift

```diff
@@ -2,22 +2,25 @@ import Foundation
 
 public struct EventServer {
 
-  static func getEvents() -> [GeneralEvent] {
-    //TODO: source events from database
-    let hypeDuration : Double = 4000000
-    let duration : Double = 4000
-    let workshopEvents : [GeneralEvent] = WorkshopManager.workshops.map{
-      (workshop: Workshop) -> GeneralEvent in
-        return GeneralEvent(title: workshop.title, time:  DateInterval(start: Date(), duration: duration),
-          tagLine: "There's Pizza", color: "purple", hypePeriod: DateInterval(start: Date(), duration: hypeDuration),
-          tags: ["tag1", "tag2"], description: Markdown("Thing"))
-    }.filter{ event in 
-      event.shouldShowAsUpdate
+  private(set) static var events: [GeneralEvent] = []
+
+  static func update() {
+    let newEvents = try? GeneralEvent.makeQuery().filter("title", .notEquals, nil).all()
+    guard let notNilEvents = newEvents else {
```

Comment:
surely:
```swift
guard let notNilEvents = newEvents {
   events = notNilEvents
}
```
is a cleaner way of doing this whole function right?

Reply:
Don't think so, but:
```
events = newEvents ?? []
```
should do the job!

Reply:
I do think we want to leave in the behaviour of defaulting to an empty array if the query doesn't return anything. That would require setting `events` to `[]` before the guard which isn't really any different to what's already there.